apps/nsqadmin/main.go

```diff
@@ -9,7 +9,7 @@ import (
 	"syscall"
 
 	"github.com/BurntSushi/toml"
-	"github.com/judwhite/go-svc/svc"
+	svc "github.com/judwhite/go-svc"
 	"github.com/mreiferson/go-options"
```

Comment:
@jehiah You know, it was the early days. Every package started with `go-` 😆

Reply:
goodness knows I have a bunch of those myself 😢

Reply:
@jehiah Sometimes I forget to mention what I'm trying to point out 🤦‍♂️ `go-options` (the line I clicked 'Comment' on) is used like `options`. It's no big deal, just pointing it out for consistency.

Reply:
@judwhite That's a good point (and consistency is a compelling argument)

What i didn't know when i wrote this (it took me too long to track this down) is that `goimports` does special case the `"go-"` prefix when handling import paths; I didn't know that.

https://github.com/golang/tools/blob/b894a3290fff7ed8373c3156460600f8216a6c2d/internal/imports/fix.go#L1128-L1151